docs: Add note about sign-in issues in troubleshooting (#95)
briefercloud/briefer: docs/DEPLOYMENT.md

If you can't figure out what's happening, please [open an issue here](https://github.com/briefercloud/briefer/issues).

</details>

<details>

>   <summary>I can access the web application, but sign-in is not working</summary>

Reviewer: ```suggestion
  <summary>I can access the web application, but the sign-in is not working</summary>
```

Nitpick

> If you're running Briefer from another machine - for example an external server, Raspbery Pi etc - make sure you have enabled HTTPS for your current setup while accessing Briefer's application on port 3000.

Reviewer: ```suggestion
If you're running Briefer in another machine, for example an external server, Raspbery Pi, etc - make sure you have enabled HTTPS for your current setup while accessing Briefer on port `3000`.
```

> In case you can't enable HTTPS or just want to try it out, you can run Briefer in an insecure manner by setting the `ALLOW_HTTP` environment variable to `true`:

Reviewer: ```suggestion
In case you can't enable HTTPS or just want to try it out, you can run Briefer over HTTP by setting the `ALLOW_HTTP` environment variable to `true`:
```

I think it's better to just say HTTP here because it might be that people will deploy a load balancer to terminate SSL in front of it and then it's not necessarily insecure. I think HTTP is precise enough and most people will understand that HTTPS > HTTP.

Wdyt?

```sh

> user@server:~$ ALLOW_HTTP=true briefer

Reviewer: ```suggestion
ALLOW_HTTP=true briefer
```

We usually don't add the `$` in front of code blocks so it's easier to copy and paste

Author: Sounds good - it just looked nicer :P

```

> This will not set the session cookie as Secure, thus allowing you to sign-in using HTTP. Note this option should not be used in production given the [security risks](https://owasp.org/www-community/vulnerabilities/Insecure_Transport).

Reviewer: ```suggestion
Note that using `ALLOW_HTTP` will _not_ set the session cookie as Secure, thus allowing you to sign in using HTTP. We don't recommend using this option production.
```

This is similar to what I suggested earlier.
